corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserMain.java

```diff
@@ -75,17 +75,23 @@ public static void main(String[] args) {
             log.info("Successfully connected to {}", singleNodeEndpoint);
 
             CorfuStoreBrowser browser = new CorfuStoreBrowser(runtime);
+            CorfuTable<CorfuDynamicKey, CorfuDynamicRecord> table;
+            String namespace = opts.containsKey("--namespace") ?
```

Comment:
let's use `Optional` here and avoid using null-s

Reply:
actually using null makes it easier to detect and fail gracefully.